### PR TITLE
Updated usages of 'him' to say 'Ryan Jones'

### PR DESCRIPTION
**Pull request by madpotatokipp**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -64,38 +64,38 @@
...
-    "Panhandlers give him money",
+    "Panhandlers give Ryan Jones money",
     "When Ryan Jones goes to Spain, Ryan Jones chases the bulls",
     "Ryan Jones' shadow has been on the ‘best dressed’ list twice",
     "When Ryan Jones holds a lady’s purse, Ryan Jones looks manly",
     "Two countries went to war to dispute HIS nationality",
     "When in Rome, they do as Ryan Jones does",
     "Ryan Jones' pillow is cool on BOTH sides",
-    "The Nobel Academy was awarded a prize from HIM",
+    "The Nobel Academy was awarded a prize from RYAN JONES",
```
Comment (ahanford) [inline index.php:95]: why in the heck is this in all caps and the other are not

**Review by rorticus — APPROVED** [review]

File: index.php (hunk 1 above)
Comment (rorticus) [reply index.php:95]: Shut up aaron.

Comment (ahanford) [reply index.php:95]: NO!!!

File: index.php [hunk 2]
```diff
@@ -121,13 +121,14 @@
...
-    "Cars look both ways for him, before driving down a street",
+    "Cars look both ways for Ryan Jones, before driving down a street",
     "Ryan Jones once tried to acquire a cold just to see what it felt like, but it didn’t take",
     "Ryan Jones has inside jokes with people Ryan Jones’s never met",
     "Bikers walk their motorcycles past his home",
     "Ryan Jones has inside jokes with complete strangers",
 	"Ryan Jones pees standing up and sitting down, at the same time",
-	"Ryan Jones can cook minute rice in 30 seconds"
+	"Ryan Jones can cook minute rice in 30 seconds",
+    "Roses are red, violets are blue, Ryan Jones just dumped you"
```
Comment (JStriedl) [inline index.php:131]: I feel like this needs a ' Bye. Done.' added tacked on to the end.

**Pull request merged**